projects/1.20/assets/veggies-delight/veggiesdelight/lang/zh_cn.json

```
@@ -1,47 +1,47 @@
 {
   "item.veggiesdelight.bellpepper": "甜椒",
   "item.veggiesdelight.smoked_bellpepper": "烟熏甜椒",
-  "item.veggiesdelight.cacciatore": "猎人肉酱",
+  "item.veggiesdelight.cacciatore": "猎人炖菜",
   "item.veggiesdelight.shakshouka": "北非蛋",
   "item.veggiesdelight.stuffed_bellpeppers": "填馅甜椒",
   "item.veggiesdelight.bellpepper_seeds": "甜椒种子",
-  "item.veggiesdelight.uncooked_mhadjeb": "生阿尔及利亚薄饼",
-  "item.veggiesdelight.mhadjeb": "阿尔及利亚薄饼",
-  "item.veggiesdelight.vegan_pizza_slice": "全素披萨切片",
+  "item.veggiesdelight.uncooked_mhadjeb": "生姆哈吉薄饼",
+  "item.veggiesdelight.mhadjeb": "姆哈吉薄饼",
+  "item.veggiesdelight.vegan_pizza_slice": "纯素披萨切片",
 
   "item.veggiesdelight.cauliflower": "花椰菜",
-  "item.veggiesdelight.cauliflower_seeds":"花椰菜种子",
+  "item.veggiesdelight.cauliflower_seeds": "花椰菜种子",
   "item.veggiesdelight.cauliflower_floret": "花椰菜花球",
-  "item.veggiesdelight.fried_cauliflower_floret": "油炸花球",
-  "item.veggiesdelight.cauliflower_patty": "花椰菜饼",
-  "item.veggiesdelight.cooked_cauliflower_patty": "熟花椰菜饼",
-  "item.veggiesdelight.cauliflower_soup": "花椰奶油浓汤",
-  "item.veggiesdelight.cauliflower_burger": "花椰菜汉堡",
+  "item.veggiesdelight.fried_cauliflower_floret": "香烤花椰菜",
+  "item.veggiesdelight.cauliflower_patty": "生花椰菜蛋饼",
+  "item.veggiesdelight.cooked_cauliflower_patty": "花椰菜蛋饼",
+  "item.veggiesdelight.cauliflower_soup": "奶油花椰菜浓汤",
+  "item.veggiesdelight.cauliflower_burger": "花椰菜素汉堡",
 
   "item.veggiesdelight.sweet_potato": "红薯",
   "item.veggiesdelight.baked_sweet_potato": "烤红薯",
-  "item.veggiesdelight.mashed_potatoes": "薯泥",
+  "item.veggiesdelight.mashed_potatoes": "奶香薯泥",

```

```suggestion
  "item.veggiesdelight.mashed_potatoes": "马铃薯泥",
```

```suggestion
  "item.veggiesdelight.mashed_potatoes": "薯泥",
```

原文即“Mashed Potatoes”，和奶香无关。加入奶油、牛奶制作土豆泥只是常用的烹饪手法。
如果严格按原版术语走就是“马铃薯泥”，不过菜肴命名确实千奇百怪的，“薯泥”也可。

---

这个既可以用马铃薯制作，也可以用该模组添加的红薯制作。事实上，这个的合成配方中用的是矿词，考虑到这种情况，在与前任翻译者沟通后（详见前面的commit），我才决定翻译成薯泥。至于奶香，这个的确是我个人的修饰。不过考虑到农夫乐事模组的官方翻译中也存在些许修饰，同时奶香并没有破坏翻译的意义，因此我建议保留。希望您告诉我您的意见。

---

> 薯泥
sweet potato也是一种potato，估计作者眼里是这样的。`薯泥`也说得通。

> 奶香
这部分我觉得属于对游戏内容望文生义的成分了。建议不要加……不过合成表里有，你加了我也不好说什么

---

的确是因为合成表里有我才加的。我觉得这样比较好一点？毕竟在MC圈中，在翻译的同时加点修饰并不算罕见。